spring-integration-http/src/main/resources/org/springframework/integration/http/config/spring-integration-http-5.0.xsd

```diff
@@ -726,6 +859,60 @@
 		</xsd:attribute>
 	</xsd:complexType>
 
+	<xsd:attributeGroup name="asyncHttpOutboundCommonAttributes">
+		<xsd:attribute name="async-rest-template" type="xsd:string">
```

Comment:
The naming is from ampq:outbound-async-gateway which use async-template to hold the AsyncRabbitTemplate. What you think?